core/src/components/textarea/textarea.tsx

```diff
@@ -15,6 +15,8 @@ import type { TextareaChangeEventDetail, TextareaInputEventDetail } from './text
 
 /**
  * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
+ *
+ * @slot label - @experimental The label text to associate with the textarea. Use the `labelPlacement` property to control where the label is placed relative to the textarea. Use this if you need to render a label with custom HTML.
```

Comment:
Is `@experimental` in the right place here? It shows up weird in the component file

Reply:
See https://github.com/ionic-team/ionic-framework/pull/27647#discussion_r1230172878